src/main/java/Duke.java

```diff
@@ -1,10 +1,83 @@
+import java.util.Scanner;
+
 public class Duke {
+
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+
+        String userInput;
+        Scanner in = new Scanner(System.in);
+        Task[] t = new Task[100];
+        int listIndex = 0;
+
+        System.out.println("____________________________________________________________");
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you?");
+        System.out.println("____________________________________________________________");
+
+        userInput = in.nextLine();
+
+        while (!(userInput.contentEquals("bye"))) {
+            if(userInput.contentEquals("list")){
+                System.out.println("____________________________________________________________");
+                System.out.println("List so far: ");
+                for(int i = 0; i < listIndex; i++) {
+                    System.out.print(i +  1);
+                    System.out.println(t[i]);
+                }
+                System.out.println("____________________________________________________________");
+                userInput = in.nextLine();
+            }
+            else {
+                if(userInput.contains("done")){
+                    int task_index = Integer.parseInt(String.valueOf(userInput.charAt(userInput.length() - 1)));
```

Comment:
Suggestion:
You could break this line down into multiple commands for greater readability